_posts/2020-11-11-hacktoberfest-sum-up.md

```diff
@@ -0,0 +1,39 @@
+---
+
+title: "Hacktoberfest 2020 sum-up"
+author: [molnarm84, huszerlbence]
+image: "../img/posts/hacktoberfest2020_sum-up.png"
+tags: [hacktoberfest, hacktoberfest2020, community, github, open source, JS, JavaScript, Cypress, end-to-end tests]
+redirect_to: https://www.sensenet.com/blog/2020-11-11-hacktoberfest-sum-up
+
+---
+
+Hacktoberfest 2020 is over, thanks for all the contributors. Check this year's stats and results.
+
+---
+
+sensenet joined #hacktoberfest celebration for the second time. This year, learned from our experiences back in 2019, we prepared a lot and created a dedicated site and much more to help contributors around the world🌍.
+
+This was a special year for us since we launched 🚀, along with hacktoberfest, our new software as a service model (called SNaaS).
+
+In the first week we had many registrations and contributors started to work on the dedicated issues. In the coming weeks the buzz chilled down, mainly because of stricter event rules and because of the fact that this year we wanted real contributions in the form of end-to-end tests written in cypress.
+Now, let's see the exact numbers.
+
+# Stats
+
+**resolved issues: 20**
+
+**winners: 13** (special thanks for those who resolved more than one issue)
+
+Since we launched SNaaS in BETA along with hacktoberfest, as part of the challenge, contributors had to register to the service and claim a free repository. During october we had 36 new repo registrations altogether. These repositories can be used in the future as part of our developer plan or upgraded if your project requires so (see our plans [here](https://www.sensenet.com/pricing)).
```

Comment:
```suggestion
Since we launched SNaaS in BETA along with Hacktoberfest, as part of the challenge, contributors had to register to the service and claim a free repository. During October we had 36 new repo registrations altogether. These repositories can be used in the future as part of our developer plan or upgraded if your project requires so (see our plans [here](https://www.sensenet.com/pricing)).
```